measure_flakiness.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# Measures the flakiness (the percentage of failures) of the given command.
+#
+# Usage: measure_flakiness.sh $COMMAND_TO_RUN
+
+RUN_COUNT=10
```

Comment:
It would be nice if `RUN_COUNT` could be an argument that the user could set.

Reply:
Done in #22